Tests/SwiftParserTest/StatementTests.swift

```diff
@@ -286,7 +291,18 @@ final class StatementTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code 'foo()' before conditional compilation clause"),
         DiagnosticSpec(locationMarker: "2️⃣", message: "all statements inside a switch must be covered by a 'case' or 'default' label", fixIts: ["insert label"]),
-      ]
+      ],
+      fixedSource: """
+        switch x {
+          foo()
```

Comment:
We should also be offering to add `case <#identifier#>:` here, like we do inside the conditional compilation clause.